src/read_pool.rs

```diff
@@ -276,8 +276,11 @@ pub fn build_yatp_read_pool<E: Engine, R: FlowStatsReporter>(
             config.min_thread_count,
             config.max_thread_count,
             std::cmp::max(
-                UNIFIED_READPOOL_MIN_CONCURRENCY,
-                SysQuota::cpu_cores_quota() as usize,
+                std::cmp::max(
```

Comment:
Why not follow the same style as in config?

Reply:
Which "style" are you referring? If you mean this parameter should be set to  cpu_quota * 10 as in the config.rs, then it is a different things. I believe that this parameter was introduced for the "auto adjustment" feature, the original author's intention was
`config.min_thread_count < config.max_thread_count < this parameter (the real upper limit)`, in case the config.max_thread_count is set to a small value. I realized that the naming and logic are confusing, and was trying to change this, but other team members were conservative about changing it since it might change the user behavior. So, I kept it untouched.